charts/opentelemetry-ebpf-instrumentation/values.yaml

```diff
@@ -0,0 +1,337 @@
+    # network:
+    #   enable: true
+    prometheus_export:
+      port: 9090
+      path: /metrics
+    ## to enable internal metrics
+    # internal_metrics:
+    #   prometheus:
+    #     port: 6060
+    #     path: /metrics
+
+## Env variables that will override configmap values
+## For example:
+##   OTEL_EBPF_INTERNAL_METRICS_PROMETHEUS_PORT: 9090
+# -- extra environment variables
+env: {}
+  # OTEL_EBPF_INTERNAL_METRICS_PROMETHEUS_PORT: 9090
+# OTEL_EBPF_TRACE_PRINTER: "text"
+
+# -- extra environment variables to be set from resources such as k8s configMaps/secrets
+envValueFrom: {}
+  #  ENV_NAME:
+  #    secretKeyRef:
+  #      name: secret-name
+#      key: value_key
+
+# -- Preconfigures some default properties for network or application observability.
+# Accepted values are "network" or "application".
+preset: application
+
+# -- Enable creation of ServiceMonitor for scraping of prometheus HTTP endpoint
+serviceMonitor:
+  enabled: false
+  # -- Add custom labels to the ServiceMonitor resource
+  additionalLabels: {}
+  # -- ServiceMonitor annotations
+  annotations: {}
+  metrics:
+    # -- ServiceMonitor Prometheus scraping endpoint.
+    # Target port and path is set based on service and `prometheus_export` values.
+    # For additional values, see the ServiceMonitor spec
+    endpoint:
+      interval: 15s
+  internalMetrics:
+    # -- ServiceMonitor internal metrics scraping endpoint.
+    # Target port and path is set based on service and `internal_metrics` values.
+    # For additional values, see the ServiceMonitor spec
+    endpoint:
+      interval: 15s
+  # -- Prometheus job label.
+  # If empty, chart release name is used
+  jobLabel: ""
+
+# -- Options to deploy the Kubernetes metadata cache as a separate service
+k8sCache:
+  # -- Number of replicas for the Kubernetes metadata cache service. 0 disables the service.
+  replicas: 0
+  # -- Enables the profile port for the Opentelemetry eBPF Instrumentation cache
+  profilePort: 0
+  ## Env variables that will override configmap values
+  ## For example:
+  ##   Opentelemetry eBPF Instrumentation_K8S_CACHE_LOG_LEVEL: "debug"
+  # -- extra environment variables
+  env: {}
+  # Opentelemetry eBPF Instrumentation_K8S_CACHE_LOG_LEVEL: "debug"
+
+  # -- extra environment variables to be set from resources such as k8s configMaps/secrets
+  envValueFrom: {}
+    #  ENV_NAME:
+    #    secretKeyRef:
+    #      name: secret-name
+  #      key: value_key
+  resources: {}
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # requests:
+    #   cpu: 100m
+  #   memory: 128Mi
+  image:
```

Comment:
TODO: need to have an offical otel image for this, or see if we actually need this